### PR TITLE
timeshift: Initial commit

### PR DESCRIPTION
**Pull request by algent-al**
Timeshift is a good tool to make full system backups.
it needs a dependency `crond` which in fact it is `dcron` [deprecated](https://dev.getsol.us/source/dcron/) from Solus Repos. Since `dcron` is blacklisted I had to rename it to `crond`.
These two files below I took them as they were in Solus Repo.
`crond/files/root.crontab`
`crond/files/run-cron`
Maybe they are not need, if so I will remove them.

About `timeshift` scheduled snapshots are not working anymore, but ondemand snapshots are working really fine. Timeshift can be used in CLI or GUI.

### CODE REVIEW

**Comment by algent-al** [comment]
Timeshift request was [here](https://dev.getsol.us/T220).

**Comment by streambinder** [comment]
I honestly think in this case the choice by the Solus core team to drop support for `dcron` is worthy and, thus, I'd rather not include it back in this repository.

On the other hand, I'd like to welcome `timeshift`.

> About timeshift scheduled snapshots are not working anymore, but ondemand snapshots are working really fine. Timeshift can be used in CLI or GUI.

So what is `dcron` dependency for? Has timeshift dropped support for scheduled tasks?

**Comment by algent-al** [comment]
Timeshift is looking for `dcron` dependency. Let me try again without it.

**Comment by algent-al** [comment]
crontab

Yes it needs it.

**Comment by algent-al** [comment]
some distro use `cronie` but I don't know how to package it.

**Comment by Justinzobel** [comment]
It uses cron for scheduling backups.

**Comment by streambinder** [comment]
Going to merge it as soon as I get Github Actions for automatic update running.
